src/features/form/util.js

```diff
@@ -5,6 +5,8 @@ const util = {
   // TODO update this next two methods to use form field and namespace instead of comment when running OpenMRS 1.11+
   getFormAndPathFromObs: (obs) => {
 
+    if (!obs.comment) { return {}; }
```

Comment:
Why should it do this? This seems like surprising behavior

Reply:
If there's no comment, it means there's no form or path... do you think it's better if it returns null?

Reply:
Ah, given what you're telling me, that seems like reasonable behavior.

Reply:
Eventually, this will support fetching the form field and namespace either from the actual form and namespace fields, or from the comment